css/index.css

```diff
@@ -1,3 +1,635 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container{
+	width: 1200px;
+	margin: auto;
+}
+nav{
+	display: flex;
+	padding: 10px;
+	width: 100%;
+	justify-content: space-around;
+	align-items:center;
+	flex-wrap: wrap;
+}
+nav a{
+	padding:10px;
+	margin:10px;
+	text-decoration: none;
+	color: gray;
+	box-sizing: border-box;
+	text-align: : center;
+	font-size:16px;
+}
+
+nav img{
+	padding: 10px;
+	margin:10px;
+}
+
+.hero-section{
+	width:100%;
+	height: 550px;
+	display: flex;
+	justify-content:space-around;
+	align-items: center;
+	flex-wrap: wrap;
+	border-bottom: 1px solid #717171
+}
+
+.hero-section .copy{
+	width: 470px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	flex-wrap: nowrap;
+}
+
+.hero-section  h1{
+	font-size: 85px;
+	text-align: center;
+	width: 300px;
+}
+
+.hero-section img{
+	width:500px;
+	height: 500px;
+}
+
+.hero-section button{
+	width: 200px;
+	font-size:16px;
+	height: 50px;
+	background: white;
+	border-style: solid;
+	border-radius: : 2px;
+	border-color: black;
+}
+
+.intro{
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+}
+
+.intro h2{
+	font-size: 24px;
+}
+.intro .features{
+	width: 50%;
+	margin: 20px;
+	padding: 20px;
+}
+.intro .about{
+	width: 50%;
+	margin:20px;
+	padding: 20px;
+}
+
+.middle-img{
+	display: flex;
+	justify-content:center;
+	align-items: center;
+	width: 100%
+}
+
+
+
+.details{
+	display: flex;
+	justify-content: space-around;
```

Comment:
The margins on details seem to be inconsistent with the rest of the page. That might be because you are using `space-around`, which would put space between the first and last elements within the flex container. Try `space-between` instead.